core/libs/commonwealth/commonwealth/settings/manager.py

```diff
@@ -0,0 +1,120 @@
+import os
```

Comment:
If using [pathlib.Path native methods](https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module), `os` in not needed.

Reply:
Perfect, thanks!